### PR TITLE
Love

### PR DESCRIPTION
**Pull request by fjlaubscher**
- updated dependencies
- cleaned up rollup config to stick to new version
- removed browser version of the component library, no one is going to use it like that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,12 @@
 {
   "name": "@leadhome/oxygen",
-  "version": "0.0.5",
+  "version": "1.0.0",
```
Comment (seagyn) [inline package.json]: Should this really be a "full" release?

Comment (fjlaubscher) [reply package.json]: Maybe not, but the npm versioning was messed up and the minor version was already taken.

Comment (seagyn) [reply package.json]: I think it should be 0.0.7?

Comment (fjlaubscher) [reply package.json]: Patches are for bug fixes, this should ideally increment a minor version.
But 0.1.0 already exists for some reason.

Comment (seagyn) [reply package.json]: I can't see a 0.1.0 on NPM so not sure where it's picking that up from.

**Pull request merged**